### PR TITLE
create k8s deployment from template

### PR DESCRIPTION
**Pull request by rpitonak**

### CODE REVIEW

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/Ugps](https://da.gd/Ugps)__
* Comment: __See details for logs__

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/NpYcT](https://da.gd/NpYcT)__
* Comment: __See details for logs__

File: tests/integration/test_k8s.py [hunk 1]
```diff
@@ -159,3 +157,44 @@ def test_list_deployments():
             finally:
                 db_deployment.delete()
                 k8s_backend.delete_namespace(namespace)
+
+
+@pytest.mark.xfail
```
Comment (jpopelka) [inline tests/integration/test_k8s.py]: What about adding some minikube install&set magic into [test-requirements.sh](https://github.com/user-cont/conu/blob/master/test-requirements.sh) ? Wouldn't that help ?

File: README.md [hunk 2]
```diff
@@ -184,6 +184,79 @@ $ python3 examples/readme_webserver.py
 
 The test passed! The logs should be easy to read, so you should have pretty good overview of what happened.
 
+# Kubernetes
+
+## Use conu with minikube locally
+
+If you want to test your images in Kubernetes locally, you will need to run kubernetes cluster on your host. We recommend to use minikube, for installation follow instructions in [minikube github repository](https://github.com/kubernetes/minikube).
+
+After just run:
```
Comment (TomasTomecek) [inline README.md]: *After that, run minikube like this:*

File: README.md [hunk 3]
```diff
@@ -184,6 +184,79 @@ $ python3 examples/readme_webserver.py
...
+```bash
+$ cat examples/k8s_deployment.py
+```
+
+```python
+from conu.backend.k8s.backend import K8sBackend
+from conu.backend.k8s.deployment import Deployment
+
+with K8sBackend() as k8s_backend:
+    namespace = k8s_backend.create_namespace()
```
Comment (TomasTomecek) [inline README.md]: please document what does this line do

File: conu/backend/k8s/service.py [hunk 4]
```diff
@@ -104,8 +104,12 @@ def get_ip(self):
         return self.spec.cluster_ip
 
     def create_in_cluster(self):
+        """
+        call Kubernetes API and create Service in cluster, raise ConuExeption if API call fail
```
Comment (TomasTomecek) [inline conu/backend/k8s/service.py]: *call Kubernetes API and create this Service in cluster, raise ConuExeption if the API call fails*

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/1WKp](https://da.gd/1WKp)__
* Comment: __See details for logs__

**Comment by dhodovsk** [comment]
[test]

**Comment by dhodovsk** [comment]
LGTM

**Pull request merged**